packages/terra-clinical-result/src/result-time-header-cell/ResultTimeHeaderCell.jsx

```diff
@@ -62,6 +62,7 @@ const ResultTimeHeaderCell = (props) => {
       className={timeHeaderCellClassnames}
     >
       <time className={dateClassnames}>{date}</time>
+      <p className={cx('hidden-space')}>{'\u00A0'}</p>
```

Comment:
Does this need to be a <p> tag?

Reply:
No I think it could be a different tag but wasn't sure what would make the most sense for this